docs/core/project-sdk/msbuild-props-desktop.md

```diff
@@ -0,0 +1,195 @@
+---
+title: MSBuild properties for Microsoft.NET.Sdk.Desktop
+description: Reference for the MSBuild properties and items that are understood by the .NET Desktop SDK which includes WPF and WinForms.
+ms.date: 02/04/2021
+ms.topic: reference
+author: adegeo
+ms.author: adegeo
+ms.custom: updateeachrelease
+no-loc: ["App.xaml", "Application.xaml", "ApplicationDefinition", "Page", "EmbeddedResource", "Compile", "None"]
+---
+# MSBuild reference for .NET Desktop SDK projects
+
+This page is a reference for the MSBuild properties and items that you use to configure Windows Forms (WinForms) and Windows Presentation Foundation (WPF) projects with the .NET Desktop SDK.
+
+> [!NOTE]
+> This article documents a subset of the MSBuild properties for the .NET SDK as it relates to desktop apps. For a list of common .NET SDK-specific MSBuild properties, see [MSBuild reference for .NET SDK projects](msbuild-props.md). For a list of common MSBuild properties, see [Common MSBuild properties](/visualstudio/msbuild/common-msbuild-project-properties).
+
+## Enable .NET Desktop SDK
+
+To use WinForms or WPF, configure your project file.
+
+### .NET 5.0
+
+WinForms and WPF projects should specify the following settings in the project file:
+
+- Target the .NET SDK `Microsoft.NET.Sdk`. For more information, see [Project files](overview.md#project-files).
+- Set the `TargetFramework` to `net5.0-windows`.
```

Comment:
```suggestion
- Set [TargetFramework](msbuild-props.md#targetframework) to `net5.0-windows`.
```

Reply:
Would you not want to still surround it with inline code?

```suggestion
- Set [`TargetFramework`](msbuild-props.md#targetframework) to `net5.0-windows`.
```

Reply:
That's fine. I don't normally format links like that, but I know some people do.

Reply:
I like formatting ;)